Code/Framework/AzToolsFramework/AzToolsFramework/Logger/TraceLogger.cpp

```diff
@@ -73,15 +73,15 @@ namespace AzToolsFramework
         AZStd::string logPath;
         StringFunc::Path::Join(logDirectory.c_str(), logFileName.c_str(), logPath);
 
-        m_logFile.reset(aznew LogFile(logPath.c_str()));
+        m_logFile.reset(aznew LogFile(logPath.c_str(), true));
```

Comment:
Should this clearing flag be optional - won't this be destructive for users with existing logs?

Reply:
Certainly. This can be configurable. Do we generally want to store and append to logs from one run to the next?

Reply:
As a default probably not, but there will be places like the server where it's needed.